src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.stories.js

```diff
@@ -1,43 +1,24 @@
 import React from 'react';
-import {storiesOf} from '@storybook/react';
+import WebexInterstitialMeeting from './WebexInterstitialMeeting';
 
-import jsonData from '../../data';
-import {WebexJSONAdapter} from '../../adapters';
-import {WebexInterstitialMeeting, WebexDataProvider} from '..';
+export default {
+  title: 'Meetings/Webex Interstitial Meeting',
+  component: WebexInterstitialMeeting,
+};
 
-// Setup for the stories
-const stories = storiesOf('Webex Interstitial Meeting', module);
-const webexAdapter = new WebexJSONAdapter(jsonData);
-const wrapperStyle = {height: '500px', width: '800px', border: '1px solid black'};
```

Comment:
The wrapper needs to stay due to the way the meetings components require a fixed sized container.